tests/NBomber.IntegrationTests/Reporting/ReportingConfigurationTests.fs

```diff
@@ -91,3 +94,77 @@ let ``withReportFileName and withReportFolder should be properly handled`` () =
         test <@ [".html"; ".csv"; ".txt"; ".md"] |> Seq.contains file.Extension  @>
         test <@ file.Length > 0L  @>
     )
+
+[<Fact>]
+let ``withoutReports should not print report files`` () =
+
+    let inMemorySink = InMemorySink()
+
+    let createLoggerConfig = fun () ->
+        LoggerConfiguration()
+            .MinimumLevel.Is(LogEventLevel.Verbose)
+            .WriteTo.Sink(inMemorySink)
+
+    let okStep = Step.create("ok step", fun _ -> task {
+        do! Task.Delay(seconds 1)
+        return Response.ok()
+    })
+
+    let scenario =
+        Scenario.create "test" [okStep]
+        |> Scenario.withoutWarmUp
+        |> Scenario.withLoadSimulations [KeepConstant(copies = 5, during = seconds 5)]
+
+    NBomberRunner.registerScenarios [scenario]
+    |> NBomberRunner.withoutReports
+    |> NBomberRunner.withLoggerConfig createLoggerConfig
+    |> NBomberRunner.run
+    |> ignore
+
+    let logEvents = inMemorySink.LogEvents |> Seq.toList
+    let reportsBuilt = logEvents |> List.exists(fun x -> x.MessageTemplate.Text = "Report.build")
+    let txtReportPrinted = logEvents |> List.exists(fun x -> x.MessageTemplate.Text = "TxtReport.print")
+    let csvReportPrinted = logEvents |> List.exists(fun x -> x.MessageTemplate.Text = "CsvReport.print")
+    let htmlReportPrinted = logEvents |> List.exists(fun x -> x.MessageTemplate.Text = "HtmlReport.print")
+    let mdReportPrinted = logEvents |> List.exists(fun x -> x.MessageTemplate.Text = "MdReport.print")
+    let consoleReportPrinted = logEvents |> List.exists(fun x -> x.MessageTemplate.Text = "ConsoleReport.print")
+
+    test <@ reportsBuilt = true @>
+    test <@ txtReportPrinted = false @>
+    test <@ csvReportPrinted = false @>
+    test <@ htmlReportPrinted = false @>
+    test <@ mdReportPrinted = false @>
+    test <@ consoleReportPrinted = false @>
+
+[<Fact>]
+let ``withoutReports should not save report files`` () =
+
+    // delete all directories with all files
+    if Directory.Exists "./no-reports" then
```

Comment:
please add some prefix for folders names like here
https://github.com/PragmaticFlow/NBomber/blob/dev/tests/NBomber.IntegrationTests/StepTests.fs#L334

it prevents concurrent access to the same folder

Reply:
there is only one test with such folder name, but anyway, fixed